### ISSUE TITLE
Load distribution for connections in server mode is not equal.

### ISSUE THREAD

**Issue by chhil** [tilln/jmeter-iso8583#25]
I don't know if this is a Jmeter concern or a plugin concern or i haven't configured it right.

I have the plugin configured to receive connections from 3 physical servers (6 connections each)

The generated ISO8583 traffic is distributing the load but most go to one server (something like 20K to one and 500 each to the other 2). I was looking for an equal distribution.

For now the workaround is to triplicate the configuration and change the listening ports and have inbound servers connect to 3 different ports.

**Comment by tilln** [comment]
Hi @chhil 
Thanks for picking this up. I haven't been able to use/test the server mode much.

For now, the plugin does not define the `send-request` configuration element, the default value of which is [`LAST`](https://github.com/jpos/jPOS/blob/v2_1_4/jpos/src/main/java/org/jpos/q2/iso/QServer.java#L170).

See also [here](https://github.com/jpos/jPOS/blob/v2_1_4/doc/src/asciidoc/ch08/qserver.adoc) for Q2 config examples:
```<send-request>LAST</send-request><!--default last connected -->```

What you would want instead I suppose is a value of 
```<send-request>RR</send-request><!-- round-robin -->```
which should lead to an equal distribution.

I've quickly put together a [snapshot release](https://github.com/tilln/jmeter-iso8583/releases/tag/1.2-SNAPSHOT) for you that's untested so perhaps you want to give this a try?
This adds a new config item "Send Method" (any ideas for a better name?) that can be set to `RR` as above.

**Comment by chhil** [comment]
Thanks will give this release a try tomorrow.
Yes the round robin one would work for me.

SendMethod<=> Connection algorithm to use for sending.
Choices in jpos are LAST (use last connection), ALL (use all connections to send the message, basically if there are 10 connections it sends the message on all 10 connections) and  RR(use round-robin ).


Your plugin has been of great help, thank you.

**Comment by chhil** [comment]
I did a quick test.

1. Put the snapshot jar in and removed the older jar from the lib/ext.
2. Had a server listening on a port and configured to send 1 transaction per second.
3. Changed send method to RR.
3. Had 3 netcat clients running from command prompt. (ncat 127.0.0.1 20000 -x 1.log). Used 2.log and 3.log to get a hexdump of messages.
Expectation was to see the log files of approximately the same size. 
Results as expected.
![image](https://user-images.githubusercontent.com/1119439/87400599-a92a0d00-c5d6-11ea-9c34-2e743ed4d19d.png)

I did run it with the send method as "last" too and the results were expected, one log file size was growing.

Thank you for the quick fix.
